.circleci/config.yml

```diff
@@ -139,6 +139,33 @@ jobs:
           command: |
             npm run-script changeset:publish
 
+  release_next:
```

Comment:
We need to also add these changes to the main branch, right?

Reply:
I don't think so, isn't it only Vault that only works on `main`?